docs/source/python_api/mlflow.models.utils.rst

```diff
@@ -0,0 +1,6 @@
+mlflow.models.utils
+===================
+.. automodule:: mlflow.models.utils
```

Comment:
Do we really want to expose `mlflow.models.utils`? Couldn't we export important functions in `mlflow.models`? Probably same for `signature`.

Reply:
Btw, I think save_example definitely shouldn't be public initially. It is only available through mlflow model log -- seems fine.

Reply:
Would recommend inlining a subset of the `save_example` documentation. We don't need to assert as many properties (e.g., 2D arrays being unsupported).

Maybe could simplify most of the paragraphs as "The given example will be converted to a Pandas DataFrame and then serialized using the Pandas split-oriented format. Bytes are base64-encoded."

Reply:
We do not have to expose mlflow.model.utils if we hide save_example.